network_monitor_udp/CMakeLists.txt

```diff
@@ -60,3 +63,13 @@ target_link_libraries(udpmonserv ${catkin_LIBRARIES})
 target_link_libraries(udpmonsink ${catkin_LIBRARIES})
 add_dependencies(udpmonsink ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
 add_dependencies(udpmonserv ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+
+install(TARGETS udpmonserv udpmonsink
+    RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+)
+catkin_install_python(PROGRAMS nodes/network_watchdog.py nodes/sample_bwtest.py nodes/udpmonclinode.py nodes/udpmonsourcenode.py
+    DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
+
+install(FILES launch_nodes.launch
+    DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/launch
+)
```

Comment:
This installs the launch file into a `launch` subfolder.
In theory this is the way to do it, but the relative file paths have to be the same between devel and install workspaces,
otherwise e.g. launch-file includes will fail with one type of workspace.

If you insist, you can move the launch file to a `launch` subfolder in the source package, too, but to preserve the current include path, I would propose to install the launch file without the subfolder.

Reply:
Good catch!  I'm not opposed to installing without a sub-folder.  I've pushed an update for this; let me know if you'd prefer a squashed commit or something.